lib/fluent/load.rb

```diff
@@ -32,6 +32,7 @@
 require 'fluent/event'
 require 'fluent/buffer'
 require 'fluent/input'
+require 'fluent/output_chain'
```

Comment:
This `require` seems not needed because 'fluent/output' requires 'fluent/output_chain'.
If any code were broken, it depends on that file.